torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -395,7 +396,11 @@ const Tensor& resize_(
     SymIntArrayRef size,
     c10::optional<MemoryFormat> optional_memory_format) {
   // Hold sizes to verify if we actually resize `self`.
+  // Explicitly copy data, since resizing can move original data
+  // and make references invalid.
   auto org_size = self.sym_sizes();
+  std::vector<c10::SymInt> org_size_copy{org_size.begin(), org_size.end()};
+  std::vector<c10::SymInt> size_copy{size.begin(), size.end()};
```

Comment:
```suggestion
  auto orig_size_copy = self.sym_sizes().vec();
  auto size_copy = size.vec();
```

Reply:
Thanks, somehow I've missed this function. Will use it.